core/src/main/scala/io/finch/internal/currentTime.scala

```diff
@@ -1,12 +1,12 @@
 package io.finch.internal
 
+import java.time.{Instant, ZoneId}
 import java.time.format.DateTimeFormatter
-import java.time.Instant
-import java.time.ZoneOffset
+
 
 object currentTime {
   private[this] val formatter: DateTimeFormatter =
-    DateTimeFormatter.RFC_1123_DATE_TIME.withZone(ZoneOffset.UTC)
+    DateTimeFormatter.ofPattern("EEE, dd MMM yyyy HH:mm:ss zzz").withZone(ZoneId.of("GMT"))
```

Comment:
+1 for using GMT for now.

I'm not enterlly sure about the locale though. Do you think we can look up a couple more places (like Play) to see what they are doing instead? Looks like [Finatra is using Locale.ENGLISH](https://github.com/twitter/finatra/blob/d46935d0da39e4fed0a528db078edf09e02e44d9/http/src/main/scala/com/twitter/finatra/http/filters/HttpResponseFilter.scala#L25). I think I'd be perfectly happy with `Locale.US`.